swagger.yaml

```diff
@@ -6221,6 +6221,11 @@ definitions:
         default: "x86_64"
         description: The CPU architecture of the image (x86_64/arm64/etc).
         x-go-custom-tag: gorm:"default:'x86_64'"
+      cpu_architectures:
```

Comment:
As this is an API change, do we want to mark as deprecated the old cpu_architecture? Or we'll keep use both for now?

Reply:
We keep using both for now because we can't drop a field anyway... How do we mark field as deprecated? I'm happy to do so but the last time I checked, the swagger version we use did not support it

Reply:
Yeah, I usually just add a comment somewhere to indicate it's deprecated. Maybe simply in the old property description.